README.md

```diff
@@ -1,3 +1,52 @@
 # State of Our Networks
 
-This repository has organizing materials for Our Networks team.
+This repository has organizing materials for the Our Networks team.
+
+## Infrastructures
+
+### Website
+
+The Our Networks [website](https://ournetworks.ca) code is on [GitHub](https://github.com/ournetworks/ournetworks.ca), which auto-deploys to a [Digital Ocean](https://digitalocean.com) Droplet using [Travis CI](https://travis-ci.org).
+
+Our DNS registrar is [Namecheap](https://namecheap.com) with name server pointed to Digital Ocean, which allows us to use its APIs to update DNS records with our CI process as needed. For example, our website is published to [Dat](https://dat.foundation) and [IPFS](https://ipfs.io), and our CI publishes the latest IPFS content hash of our website as a TXT record on `ipfs-website.ournetworks.ca`.
+
+### Email
+
+Our Networks is a member of [May First](https://mayfirst.coop), which we rely on for email forwarding and our low-traffic [mailing list](https://lists.mayfirst.org/mailman/listinfo/ournetworks) services, among other infrastrusture things. 
+
+Our email addresses are as follow.
+
+| Email address | Purpose                                                                                                          |
+|:--------------|:-----------------------------------------------------------------------------------------------------------------|
+| billets@      | Coordinate community billets                                                                                     |
+| coc@          | Code of Conduct reporting monitored by organizer on CoC duty                                                     |
+| orga@         | Main address used for general communications and account registrations that has a forward list to all organizers |
+
+### Financial accounts
```

Comment:
Would also not yoke us to BPT in this section. I think we could point to service inventory, describe types of banking and Alterna + practices.

Would also link to 2018/2019 examples where we put banking things (e.g. invoices)